ffs/pb/ffs.proto

```diff
@@ -173,11 +173,11 @@ message InfoReply {
    InstanceInfo info = 1;
 }
 
-message WatchRequest {
+message WatchJobRequest {
```

Comment:
Sorry, nit here, but maybe should be plural `WatchJobsRequest` and below, `WatchJobsReply`

Reply:
Agree! will fix